python/build-requirements.txt

```diff
@@ -29,4 +29,7 @@ testfixtures==6.17.1
 pyspark==3.0.1
 
 #calibration
-tensorflow-probability==0.15.0
\ No newline at end of file
+tensorflow-probability==0.15.0
+
+# DAG visualization
+pygraphviz==1.7
```

Comment:
@arvindsrikantan Do we need this in the `build-requirements.txt`? I forget, but I think this is only used for tests.

Reply:
The circle CI build will need pygraphviz too for running the test (from what I understand)

Reply:
None of the tests actually use `pygrahviz`, right?